src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,111 @@
+@import "vars";
+
 @mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
-  &:hover {
-    #{$_property}: $_toValue;
+  @media (pointer: fine) {
+    transition: #{$_property} $animation-duration;
+    &:hover {
+      #{$_property}: $_toValue;
+    }
   }
 }
+
+@mixin increase-hover-scale {
+  @include hover(transform, scale(1.05));
+}
+
+@mixin for-tablet {
+  @media (min-width: $tablet-width) {
+    @content;
+  }
+}
+
+@mixin for-desktop {
+  @media (min-width: $desktop-width) {
+    @content;
+  }
+}
+
+@mixin not-for-desktop {
+  @media (max-width: calc(#{$desktop-width} - 1px)) {
+    @content;
+  }
+}
+
+@mixin set-smartphone-grid {
+  @include set-grid(2, 50%);
+}
+
+@mixin set-smartphone-height-and-grid {
+  @include set-height($smartphone-width, 500px);
+
+  @include set-smartphone-grid;
+}
+
+@mixin set-tablet-grid {
+  @include set-grid(
+    $tablet-grid-columns,
+    $tablet-grid-coumn-width
+  );
+}
+
+@mixin set-tablet-height-and-grid() {
+  justify-content: space-between;
+
+  @include set-height($tablet-width, 800px);
+
+  @include set-tablet-grid;
+}
+
+@mixin set-desktop-grid {
+  @include set-grid(
+    $desktop-grid-columns,
+    $desktop-grid-coumn-width
```

Comment:
There seems to be a typo in the variable name. It should be '$desktop-grid-column-width' instead of '$desktop-grid-coumn-width'.

Reply:
I fixed typo.